tests/dpdk_l3em_indirect_counter.py

```diff
@@ -29,229 +30,176 @@
 from ptf.testutils import *
 
 # framework related imports
-import common.utils.log as log
 import common.utils.p4rtctl_utils as p4rt_ctl
 import common.utils.test_utils as test_utils
-from common.utils.config_file_utils import get_config_dict, get_gnmi_params_simple
-from common.utils.gnmi_ctl_utils import gnmi_ctl_set_and_verify
 from common.lib.telnet_connection import connectionManager
+from common.utils.config_file_utils import (
+    get_config_dict,
+    get_gnmi_params_simple,
+)
+from common.utils.gnmi_ctl_utils import gnmi_ctl_set_and_verify, gnmi_set_params, ip_set_ipv4
 
+class IPDK_L3EM_INDIRECT_COUNTER_RESET(BaseTest):
```

Comment:
reset is a separate test case

Reply:
adding dpdk_l3em_indirect_counter_reset.py